src/quo2/components/selectors/reactions/view.cljs

```diff
@@ -0,0 +1,26 @@
+(ns quo2.components.selectors.reactions.view
+  (:require [quo2.components.reactions.resource :as reactions.resource]
+            [quo2.components.selectors.reactions.style :as style]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]))
+
+(defn view
+  [_ {:keys [start-pressed?]}]
+  (let [pressed? (reagent/atom start-pressed?)]
+    (fn [emoji
```

Comment:
About the function signature, why are we receiving the emoji as a separate parameter?

I thnk it'd be more consistent if we receive everything inside a single parameter map,